nucleus/dataset.py

```diff
@@ -1016,7 +1016,7 @@ def get_scene(self, reference_id: str) -> Scene:
             :class:`Scene<LidarScene>`: A scene object containing frames, which
             in turn contain pointcloud or image items.
         """
-        return Scene.from_json(
+        return LidarScene.from_json(
```

Comment:
Should the backend return something so we know what type of scene to instantiate? This change means we can't get anything EXCEPT lidar scenes, which is not everything right?

Reply:
Right now the only possible type of scene is LidarScene, and we intend to extend this to support videos later on.  Inclined to keep it this way for now, and revisit when we implement general video scene support